cost-analyzer/values.yaml

```diff
@@ -304,6 +304,10 @@ global:
 ## Provide a full name override option for the chart.
 # fullnameOverride: ""
 
+## Configure Chart Labels
+# chartLabels:
```

Comment:
kept the naming similar to what we have in prometheus helm charts https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/_helpers.tpl#L81-L93

Reply:
Good context, thank you!